tendrl/commons/jobs/__init__.py

```diff
@@ -51,9 +51,7 @@ def _run(self):
 
                         # Job routing
                         if raw_job.get("payload", {}).get("tags", []):
-                            NS.node_context = NS.node_context.load()
-                            tags = json.loads(NS.node_context.tags)
-                            if set(tags).isdisjoint(
```

Comment:
There's no exception just that the node-monitoring will not execute any job. Because, NodeContext is loaded from a path in etcd that is not written to by the node-monitoring and hence node-monitoring corresponding tags won't be there in that path and hence when loaded from etcd this tag won't be present and hence node-monitoring never executes any job targeted for it with specific tag "tendrl/monitoring" or "tendrl/node-monitoring" (and instead only executes those jobs that are meant for node-agent)